packages/@aws-cdk/aws-amplify/README.md

```diff
@@ -208,3 +208,14 @@ const amplifyApp = new amplify.App(stack, 'App', {
   ],
 });
 ```
+
+## Deploying Assets
+
+`sourceCodeProvider` is optional; when this is not specified the Amplify app can be deployed to using `.zip` packages. The `addAssetDeployment` branch method can be used to deploy S3 assets to Amplify as part of the CDK:
+
+```ts
+const asset = new assets.Asset(this, "SampleAsset", {});
+const amplifyApp = new amplify.App(this, 'MyApp', {});
+const branch = amplifyApp.addBranch("dev");
+branch.addAssetDeployment(asset);
```

Comment:
I know I suggested this `addAssetDeployment()` but thinking more about this, can we add more than one asset deployment to a branch? I don't think so. So maybe it should be moved to a construction prop in `Branch`/option in `addBranch()`?

Reply:
Sounds good; ammended.

Reply:
You need to update the README for the latest change.

Reply:
Done